README.md

```diff
@@ -12,7 +12,7 @@ From: https://boardgamegeek.com/boardgame/100480/freedom
 
 ### Rules:
 
```

Comment:
Pay attention with name refactoring

Reply:
?

Reply:
README:
- From: Board: Freedom is played on a 10×10 square board. Beginners can try the game on a 8×8 board. Other board sizes may be also used.
- To: it.units.sdm.project.core.FreedomBoard: Freedom is played on a 10×10 square board. Beginners can try the game on a 8×8 board. Other board sizes may be also used.

Reply:
I didn't select the proper lines, sorry